contracts/test/forkMainnet/AMMWrapper/CollectFee.t.sol

```diff
@@ -42,6 +42,53 @@ contract TestAMMWrapperCollectFee is TestAMMWrapper {
         feeCollectorMakerAsset.assertChange(int256(fee));
     }
 
+    function testCollectFeeWithWETH() public {
+        uint256 feeFactor = 100;
+        AMMLibEIP712.Order memory order = DEFAULT_ORDER;
+        order.makerAssetAddr = ETH_ADDRESS;
+        order.makerAssetAmount = 0.001 ether;
```

Comment:
Maker asset amount would be overridden at later, maybe we could skip this assignment to avoid confusion?

Reply:
Fixed in e0efdc2